code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -78,23 +78,26 @@
 				var/mob/living/carbon/human/H = A
 				if(H.check_shields(src, 0, "the [name]", attack_type = LEAP_ATTACK))
 					blocked = 1
+				pounce_cooldown = world.time + pounce_cooldown_time
 			if(!blocked)
 				L.visible_message("<span class ='danger'>[src] pounces on [L]!</span>", "<span class ='userdanger'>[src] pounces on you!</span>")
 				if(ishuman(L))
 					var/mob/living/carbon/human/H = L
-					H.apply_effect(5, WEAKEN, H.run_armor_check(null, "melee"))
+					H.apply_effect(3, WEAKEN, H.run_armor_check(null, "melee"))
 				else
-					L.Weaken(5)
+					L.Weaken(3)
 				sleep(2)//Runtime prevention (infinite bump() calls on hulks)
 				step_towards(src,L)
+				pounce_cooldown = world.time + pounce_cooldown_time
 			else
-				Weaken(2, 1, 1)
+				Weaken(1, 1, 1)
 
 			toggle_leap(0)
 			pounce_cooldown = world.time + pounce_cooldown_time
 		else if(A.density && !A.CanPass(src))
 			visible_message("<span class ='danger'>[src] smashes into [A]!</span>", "<span class ='alertalien'>[src] smashes into [A]!</span>")
-			Weaken(2, 1, 1)
+			Weaken(1, 1, 1)
+		pounce_cooldown = world.time + pounce_cooldown_time
```

Comment:
I dont think you need to set the `pounce_cooldown` 3 times in this proc. surely just doing it at the end here would work.